roles/pulp_common/scripts/move-artifacts-to-media-root.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# This script moves a directory passed in the first parameter to the location passed in as a second
+# parameter. It then creates a symlink from the old location to the new. When SELinux is enabled,
+# SELinux contexts are preserved.
+
+# Fail if the mv or ln fail
+set -e
+
+mv -Z $1 $2
```

Comment:
Why do you add `-Z` if you also call `restorecon` later on? Feels redundant but I may be missing something.

Reply:
restorecon is only called on the symlink. The symlink is created with the wrong context.

Reply:
I wonder what the performance impact of `mv -Z` is. If it needs to touch every file, it'll be expensive. On the other hand, it's probably not recursive. Can you enlighten me since I don't recall ever using `mv -Z` :)

Reply:
mv only operates on a single file/directory. So I would not expect it to do anything recursively.

Reply:
Fair enough. The only thing I would worry about is that on a system where selinux is disabled, the `-Z` parameter would fail. However, both on my Debian 10 and Gentoo systems without SELinux it appears to do the right thing and not fail. On Gentoo coreutils wasn't compiled especially with SELinux support so it appears to be portable enough that you can assume `-Z` is accepted on Linux systems in general. Thanks for entertaining this knowledge side step.

Reply:
Yeah, I also had the same concerns. I tested it on Debian 10 also.